pom.xml

```diff
@@ -312,6 +313,20 @@
                 </exclusions>
             </dependency>
 
+            <!-- Swagger -->
+            <dependency>
+                <groupId>io.springfox</groupId>
+                <artifactId>springfox-swagger-ui</artifactId>
+                <version>${swagger.version}</version>
+                <scope>compile</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.springfox</groupId>
+                <artifactId>springfox-swagger2</artifactId>
+                <version>${swagger.version}</version>
+                <scope>compile</scope>
+            </dependency>
+
```

Comment:
See comments above. Move this to the compile section and preserve alphabetical order so we don't end up with duplicates.